packages/breadcrumbs/package.json

```diff
@@ -19,12 +19,11 @@
   "publishConfig": {
     "registry": "https://artifactory.availity.com/artifactory/api/npm/local-npm/"
   },
-  "dependencies": {
-    "prop-types": "^15.5.8"
-  },
   "devDependencies": {
+    "prop-types": "^15.5.8",
```

Comment:
This should stay as a dependency and not a devDependency per: https://www.npmjs.com/package/prop-types#how-to-depend-on-this-package

Also it is important to understand that `prop-types` is a legit dependency, it _is_ required to run this package as the code imports prop-types and utilizes it. With this change, if a project failed to include their own prop-types dependency it would break.

Reply:
Thank you for clarifying! It always seemed like one of those development only things to me, but that makes sense.
Also whats up Evan